composer.json

```diff
@@ -16,13 +16,13 @@
     },
     "require": {
         "php": "^8.1",
-        "laravel/framework": "^v9.0",
-        "lenderspender/laravel-enums": "^1.1",
+        "laravel/framework": "^v10.0",
```

Comment:
The `v` prefix shouldn't be here, right?

Reply:
Can you tell me why you think this?

Reply:
There's never a `v` in composer dependencies.

https://getcomposer.org/doc/articles/versions.md#:~:text=Note%20that%20Composer%20automatically%20removes%20the%20%27v%27%20prefix%20in%20the%20actual%20tagname%20to%20get%20a%20valid%20final%20version%20number.